cs.json

```diff
@@ -1,9 +1,9 @@
 {
 	"setting": {
 		"options": "Možnosti",
-		"plugin": "Doplněk",
-		"builtin-plugins": "Vestavěné doplňky",
-		"plugin-options": "Možnosti doplňků",
+		"plugin": "Plugin",
```

Comment:
Why did you overwrite the previous translation with the english version?

Reply:
I didn't find `Doplněk` to be self explanatory/fitting term, there is no termbase or active chat to talk about it.
Overall cs.json is pretty outdated, so I feel free to change it to my image, until someone comes and chat about these terms. @joethei